eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Constrained/Spec/Utxo.hs

```diff
@@ -3,13 +3,11 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE ImportQualifiedPost #-}
 {-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE TypeApplications #-}
 
 -- | Specs necessary to generate, environment, state, and signal
 -- for the UTXO rule
-module Test.Cardano.Ledger.Constrained.V2.Conway.UTXO where
+module Test.Cardano.Ledger.Conway.Constrained.Spec.Utxo where
```

Comment:
FYI. Module structure is fine, eg: `Test.Cardano.Ledger.Conway.Constrained.Spec.Utxo` it just should live in the conformance package

Reply:
c.f. my comment above - could we perhaps also move the `Test.Cardano.Ledger.STS` tests to the conformance package? That way working on the skeleton part of this (getting the STS rules to execute) wouldn't require jumping too much between the packages?

Reply:
I'll try to move stuff out of `cardano-ledger-test` and put it into conway testlib and `cardano-ledger-conformance`.